file-formats/dobj/type/zif_aff_dobj_v1.intf.abap

```diff
@@ -0,0 +1,61 @@
+INTERFACE zif_aff_dobj_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Structure Definition</p>
+    "! Structure Definition
+    BEGIN OF ty_structure_definition,
+      "! <p class="shorttext">Structure Definition</p>
+      "! Parent Table
+      "! $required
+      record_no       TYPE n LENGTH 4,
+      "! <p class="shorttext">Parent Table</p>
+      "! Parent Table
```

Comment:
Try to keep descriptions in sentence style. This is also relevant for other places in your interface

```suggestion
      "! Parent table
```